src/Peering/Peering/help/Update-AzPeering.md

```diff
@@ -64,7 +64,7 @@ Sets the Md5 Authentication Key
 
 ### Example 2: Update UseForPeeringService
 ```powershell
-Update-AzPeering -ResourceGroupName rg1 -Name ContosoPeering -UseForPeeringService $true
+Update-AzPeering -ResourceGroupName rg1 -Name ContosoPeering
```

Comment:
delete the entire example

Reply:
Why delete the entire example?

Reply:
Without "UseForPeeringService" parameter, this example is meaningless. It updates nothing.